finalPrep/annotations.js

```diff
@@ -1,42 +1,59 @@
 // Add your annotations, line by line, to the code below using code comments.
 // Use the double // to create a new comment
 
-//Build a Bear
+//Build a Bear //The buildABear function takes five arguments and assigns various values to four variables
 function buildABear(name, age, fur, clothes, specialPower) {
+// A string is assigned to the greeting variable, which interpolates a greeting message using the `name` parameter
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+// An array is assigned to the demographics variable with the `name` and `age` parameters as the two elements
   var demographics = [name, age];
+// A string is assigned to the powerSaying variable, which concatenates a message using the specialPower parameter
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+  // An object is assigned to the builtBear variable
   var builtBear = {
+    // basicInfo is assinged the value of the demogrphics variable, which is an array
     basicInfo: demographics,
+    // clothes is assigned a dynamic value of the user input to the `clothes` parameter
     clothes: clothes,
+    // exterior is assigned a dynamic value of the user input to the `fur` parameter
```

Comment:
Nice work tracking where these values come from- this can be very important to find in code you are looking at for the first time.